lib/Elevate/Blockers/Repositories.pm

```diff
@@ -121,21 +121,18 @@ sub _yum_is_stable ($self) {
     if ( opendir( my $dfh, '/var/lib/yum' ) ) {
         my @transactions = grep { m/^transaction-all\./ } readdir $dfh;
         if (@transactions) {
-            ERROR('There are unfinished yum transactions remaining. Please address these before upgrading. The tool `yum-complete-transaction` may help you with this task.');
-            my $id = ref($self) . '::YumUnfinishedTransactions';
+            WARN('There are unfinished yum transactions remaining.');
 
-            $self->has_blocker(
-                'There are unfinished yum transactions remaining. Please address these before upgrading. The tool `yum-complete-transaction` may help you with this task.',
-                info => {
-                    name         => $id,
-                    error        => 'YUM has unfinished transactions',
-                    transactions => join( "\n", sort @transactions ),
-                },
-                blocker_id => $id,
-                quiet      => 1,
-            );
-
-            return 0;
+            if ( $self->is_check_mode() ) {
+                INFO('We will attempt to clear the unfinished yum transactions upon upgrade.');
```

Comment:
Maybe add the message from the old blocker so they know?

Reply:
The tool `yum-complete-transaction` may help you repair this in advance if you prefer to review it.

Reply:
WARN('Unfinished yum transactions detected. yum-complete-transactions will display these issues. Otherwise, elevate will execute /usr/sbin/yum-complete-transaction --cleanup-only on start');